src/data_info/services/sharing.clj

```diff
@@ -142,8 +144,9 @@
         :else                           (skip-share unshare-with fpath :not-shared)))
 
 (defn anon-readable?
-  [cm p]
-  (is-readable? cm (cfg/anon-user) p))
+  [irods p]
+  (let [perm (rods/permission irods (cfg/anon-user) (cfg/irods-zone) p)]
+    (delay (contains? #{:read :write :own} @perm))))
```

Comment:
The manifest is the only thing that uses `anon-readable?`, so I rewrote it to work using clj-irods rather than clj-jargon.